python/mxnet/gluon/data/vision/transforms.py

```diff
@@ -184,26 +184,44 @@ def forward(self, x):
         return image.imresize(x, *self._args)
 
 
-class RandomHorizontalFlip(HybridBlock):
-    """Randomly flip the input image horizontally with a probability
+class FlipLeftRight(HybridBlock):
```

Comment:
we don't need this for now. It's unclear when you would want to always flip

Reply:
fixed